backup_and_restore/application_backup_and_restore/oadp-intro.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: ASSEMBLY
+[id="oadp-introduction"]
+= Introduction to {oadp-full}
+include::_attributes/common-attributes.adoc[]
+:context: oadp-api
+:namespace: openshift-adp
+:local-product: OADP
+
+toc::[]
+
+The {oadp-first} product safeguards customer applications on the {product-title}. OADP provides disaster recovery protection for {product-title} applications, application-related cluster resources, persistent volumes, and internal images.
+
+OADP can be used to back up containerized applications and virtual machines (VMs).
+
+OADP cannot be a disaster recovery solution for *etcd* or OpenShift Operators. For more information on etcd, see xref:../../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[Backing up etcd].
+
+OAP provides the following APIs to enable multiple approaches to customize a backup to avoid backing up inappropriate resources by using namespaces or labels.
```

Comment:
```suggestion
OADP provides the following APIs to enable multiple approaches to customize a backup to avoid backing up inappropriate resources by using namespaces or labels.
```